src/Mezzio/Storageless/SessionThatIsIdentifierAware.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PSR7Sessions\Mezzio\Storageless;
+
+use Mezzio\Session\SessionIdentifierAwareInterface;
+use Mezzio\Session\SessionInterface;
+
+interface SessionThatIsIdentifierAware extends SessionInterface, SessionIdentifierAwareInterface {}
```

Comment:
IMO worth putting this in the test suite only

Reply:
Moved it to `PSR7SessionsTest\Mezzio\Storageless;`